### PR TITLE
[Remove immediate = true from lima Part I] LPS-168427

### PR DESCRIPTION
**Pull request by yuhai**
@dantewang

### CODE REVIEW

**Comment by yuhai** [comment]
ci:test:startup

File: modules/apps/redirect/redirect-service/src/main/java/com/liferay/redirect/internal/messaging/CheckRedirectNotFoundEntriesMessageListener.java [hunk 1]
```diff
@@ -47,7 +46,7 @@
  */
 @Component(
 	configurationPid = "com.liferay.redirect.internal.configuration.RedirectConfiguration",
-	service = MessageListener.class
+	service = {}
```
Comment (dantewang) [inline modules/apps/redirect/redirect-service/src/main/java/com/liferay/redirect/internal/messaging/CheckRedirectNotFoundEntriesMessageListener.java:49]: @yuhai You say `The commit will use _schedulerEngineHelper.register() to register MessageListener service.` but I didn't see this change?

Comment (dantewang) [reply modules/apps/redirect/redirect-service/src/main/java/com/liferay/redirect/internal/messaging/CheckRedirectNotFoundEntriesMessageListener.java:49]: @yuhai Oh, it's already there. Then you should change your commit message.

**Comment by zhangyan0701** [comment]
ci:test:lima

**Comment by zhangyan0701** [comment]
Possible regression:
Two tests in com.liferay.redirect.internal.messaging.test.CheckRedirectNotFoundEntriesMessageListenerTest failed:
```
[beanshell] Waiting for service com.liferay.portal.kernel.messaging.MessageListener component.name=*.CheckRedirectNotFoundEntriesMessageListener for field com.liferay.redirect.internal.messaging.test.CheckRedirectNotFoundEntriesMessageListenerTest._checkRedirectNotFoundEntriesMessageListener
[beanshell] Waiting for service com.liferay.portal.kernel.messaging.MessageListener component.name=*.CheckRedirectNotFoundEntriesMessageListener for field com.liferay.redirect.internal.messaging.test.CheckRedirectNotFoundEntriesMessageListenerTest._checkRedirectNotFoundEntriesMessageListener
[beanshell] Waiting for service com.liferay.portal.kernel.messaging.MessageListener component.name=*.CheckRedirectNotFoundEntriesMessageListener for field com.liferay.redirect.internal.messaging.test.CheckRedirectNotFoundEntriesMessageListenerTest._checkRedirectNotFoundEntriesMessageListener
[beanshell] Waiting for service com.liferay.portal.kernel.messaging.MessageListener component.name=*.CheckRedirectNotFoundEntriesMessageListener for field com.liferay.redirect.internal.messaging.test.CheckRedirectNotFoundEntriesMessageListenerTest._checkRedirectNotFoundEntriesMessageListener
[beanshell] Waiting for service com.liferay.portal.kernel.messaging.MessageListener component.name=*.CheckRedirectNotFoundEntriesMessageListener for field com.liferay.redirect.internal.messaging.test.CheckRedirectNotFoundEntriesMessageListenerTest._checkRedirectNotFoundEntriesMessageListener
[beanshell] Waiting for service com.liferay.portal.kernel.messaging.MessageListener component.name=*.CheckRedirectNotFoundEntriesMessageListener for field com.liferay.redirect.internal.messaging.test.CheckRedirectNotFoundEntriesMessageListenerTest._checkRedirectNotFoundEntriesMessageListener
     [exec] com.liferay.redirect.internal.messaging.test.CheckRedirectNotFoundEntriesMessageListenerTest > testDeletesEntriesOverflowing1000Elements FAILED
     [exec]     java.lang.IllegalStateException: Timed out while waiting for service com.liferay.portal.kernel.messaging.MessageListener component.name=*.CheckRedirectNotFoundEntriesMessageListener
... ...
     [exec] com.liferay.redirect.internal.messaging.test.CheckRedirectNotFoundEntriesMessageListenerTest > testDeletesEntriesOlderThan30Days FAILED
     [exec]     java.lang.IllegalStateException: Timed out while waiting for service com.liferay.portal.kernel.messaging.MessageListener component.name=*.CheckRedirectNotFoundEntriesMessageListener
```
Full console:
https://testray.liferay.com/reports/production/logs/2022-11/test-1-32/test-portal-acceptance-pullrequest(master)/7008/modules-integration-mysql57-jdk8/0/18/jenkins-console.txt.gz?authuser=0

**Pull request closed without merging**